### DISCUSSION TITLE
Drawing glitch

### DISCUSSION THREAD

**Discussion by Coffeemoth** [keiwando/toonsquid#485] [General]
Hey, I recently moved from FlipaClip to toon squid but I find this a massive problem and I don’t know what to do to fix it. Every time I try to get rid of the missing drawing frame it always comes back please help ;(

![IMG_0276](https://github.com/keiwando/toonsquid/assets/153572884/b780d42d-d069-4502-b2de-feaf24f15ca7)

https://github.com/keiwando/toonsquid/assets/153572884/7ac32e54-6586-433a-b76d-8d35a3b5151b

**Comment by keiwando** [comment]
Hi! This seems to be a symbol layer. Is there a drawing on the first frame in the referenced animation clip of the symbol? If there isn't and you can't edit that clip for some reason, then you could set the [`Time`](https://toonsquid.com/handbook/symbols/symbol_layer/#time) property of the symbol layer to 2 to make the symbol start playing from frame 2 of the clip. You could also try the [Split Drawing](https://toonsquid.com/handbook/interface/timeline/#drawing-actions) action, but that might also mess with your keyframes.

I'm mostly guessing here since it's hard to tell from the video what the exact issue is. You seem to only be dragging the retiming handle, which just edits the start time of that drawing, so it's expected that this wouldn't get rid of that empty frame. If the things I mentioned above don't fix the issue, feel free to send me a copy of the project file to contact@toonsquid.com so I can take a look inside and see what exactly the issue is caused by.

**Reply by Coffeemoth** [reply]
Thank you so so much for giving me closure your amazing!! ^^
